src/main/java/dev/lu15/voicechat/VoiceChatImpl.java

```diff
@@ -46,19 +42,43 @@ final class VoiceChatImpl implements VoiceChat {
     private final @NotNull VoiceServer server;
     private final int port;
     private final @NotNull String publicAddress;
+    private final int mtu;
+    private final Codec codec;
```

Comment:
Please add the necessary nullability annotations to fields and constructor parameters.